src/services/api/EmployeeAPI.ts

```diff
@@ -0,0 +1,9 @@
+import { defineStore } from "pinia";
+import http from "@/services/axios/axios";
+export const apiStore = defineStore("API", {
+  actions: {
+    getAllEmployee(url: string, page: number, limit: number) {
+      return http.get(`/${url}?page=${page}&limit=${limit}`);

```

Comment:
dùng đồng bộ axiosService. Param thì dùng object ko sử dụng string
```suggestion
      return axiosService.get(url, params);

```